playbooks/roles/setup-dcmm/files/cache_docker_images.yml

```diff
@@ -1,17 +1,17 @@
 ---
-
 - name: Cache docker images
   hosts: nodes
   remote_user: ec2-user
   vars:
     state: 'present'
   tasks:
     - name: Docker pull images
-      shell: /usr/bin/docker pull {{ item }}
-      become: true
+      docker_image:
+        name: "{{ item }}"
+      become: true        
       with_items:
-       - kubernetes/guestbook
```

Comment:
This image doesn't exist anymore in dockerhub plus we don't reference it in the dc metro map lab